prepare-iso.sh

```diff
@@ -142,23 +142,37 @@ function installerExists()
 # See if we can find either the ElCapitan or the 10.12 installer.
 # If successful, then create the iso file from the installer.
 #
+# The first entry is currently just and educated guess and may need
+# updating when Sierra is released.
 
-installerExists "Install 10.12 Developer Preview.app"
+installerExists "Install macOS Sierra.app"
 result=$?
 if [ ${result} -eq 0 ] ; then
-  createISO "Install 10.12 Developer Preview.app" "Sierra"
+  createISO "Install macOS Sierra.app" "Sierra"
 else
-  installerExists "Install OS X El Capitan.app"
+  installerExists "Install macOS Sierra Public Beta.app"
   result=$?
   if [ ${result} -eq 0 ] ; then
-    createISO "Install OS X El Capitan.app" "ElCapitan"
+    createISO "Install macOS Sierra Public Beta.app" "Sierra"
```

Comment:
I think until we hit a full release, I'm going to leave this PR unmerged, just because it will definitely need some changes once the full release comes out. Otherwise, I could consider creating a separate 'beta' branch that could run the latest beta, and the master branch would run the latest stable macOS release.